Colours.txt

```diff
@@ -6,6 +6,7 @@
 #define centrelinecolour 15790135
 #define col_gate 3953980
```

Comment:
This col_gate definition is not in alphabetical order, so when the compiler is run and the colours are put into alphabetical order the comment ends up in the wrong place:
![PR#4722 Problem](https://user-images.githubusercontent.com/61326713/236698199-c46869d6-dc10-42a8-90a0-4aef057a6ab7.png)

So just moving the col_gate line below the new preserved comment line should fix it I think.